src/tests/erroring/function_nonfailable_with_typed_failable_return.ab

```diff
@@ -1,5 +1,5 @@
 // Output
-// Non-failable functions cannot return a Failable type
+// Infallible functions must not have a '?' after the type name
```

Comment:
Consider changing to "Functions that cannot fail must not have a '?' after the type name" for consistency with the other error message.

Reply:
wouldn't it confuse the user though? i would definetely mistake them for each other when doing long coding sessions

- Functions that cannot fail must not have a '?' after the type name
- Functions that can fail must not have a '?' after the type name

Reply:
I'm not sure users are going to get confused by this, but I'm not going to argue the case.

Reply:
@b1ek @hdwalters How about we use these:
- Infailable functions must not have a '?' after the type name
- Failable functions must not have a '?' after the type name